server/server/src/cold_chain/sensor.rs

```diff
@@ -58,6 +58,13 @@ pub async fn put_sensors(
         Err(error) => return HttpResponse::InternalServerError().body(format!("{:#?}", error)),
     };
 
+    for result in &results {
+        if let Err(e) = result {
+            error!("Error upserting sensors {:#?}", e);
+            return HttpResponse::InternalServerError().body(format!("{:#?}", e));
+        }
```

Comment:
I updated this, but then put it back. Because I'm calling a macro called `error` it's weird to use a variable with the name name to pass into it. It does work but I think it's bit confusing.